common/scala/src/main/scala/whisk/core/entity/EntityPath.scala

```diff
@@ -170,6 +170,7 @@ protected[core] class EntityName private (val name: String) extends AnyVal {
     def toJson = JsString(name)
     def toPath: EntityPath = EntityPath(name)
     def addPath(e: EntityName): EntityPath = toPath.addPath(e)
+    def addPath(e: Option[EntityName]): EntityPath = e map { toPath.addPath(_) } getOrElse toPath
```

Comment:
No but it's going to exist somewhere (e.g. Meta.api), might as well put it here for convenience.